src/components/general/EntryCommentModal/Thread/index.js

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { _cs } from '@togglecorp/fujs';
+
+import {
+    RequestClient,
+    requestMethods,
+} from '#request';
+import ListView from '#rscv/List/ListView';
+import PrimaryButton from '#rsca/Button/PrimaryButton';
+import _ts from '#ts';
+
+import CommentFaram from '../CommentFaram';
+import Comment from './Comment';
+
+import styles from './styles.scss';
+
+const EmptyComponent = () => null;
+
+const propTypes = {
+    comments: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+    members: PropTypes.array, // eslint-disable-line react/forbid-prop-types
+    entryId: PropTypes.number,
+    threadId: PropTypes.number,
+    className: PropTypes.string,
+    // eslint-disable-next-line react/forbid-prop-types
+    commentCreateRequest: PropTypes.object.isRequired,
+    onAdd: PropTypes.func,
+    onEdit: PropTypes.func,
+    onDelete: PropTypes.func,
+    isResolved: PropTypes.bool,
+    currentEdit: PropTypes.string,
+    onCurrentEditChange: PropTypes.func,
+};
+
+const defaultProps = {
+    currentEdit: undefined,
+    isResolved: false,
+    entryId: undefined,
+    threadId: undefined,
+    className: undefined,
+    onCurrentEditChange: () => {},
```

Comment:
Why not make these methods mandatory?

Reply:
This is also used for resolved threads and they are not provided with handlers

Reply:
Its difficult to check